tf_patches/cpu_dynamic_shape.diff

```diff
@@ -1,61 +0,0 @@
-# Required for dynamic shape on TFRT:CPU. Remove when we sync past https://github.com/tensorflow/tensorflow/commit/7f045484290f57ec3dc91877af77de39275a4f78
-diff --git a/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc b/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc
-index f23bbc13e8c..25f68ffab74 100644
---- a/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc
-+++ b/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc
```

Comment:
@will-cromar Can we remove this diff since now we moved the pin?

Reply:
Yes

Reply:
this file has already been deleted in this pr..